common/pop_types/cwp_landlords.txt

```diff
@@ -6,7 +6,7 @@
 		125
 	}
 	strata = rich
-	start_quality_of_life = 18
+	start_quality_of_life = 15
```

Comment:
A landlord should have worse QoL than an Investor.

Reply:
doesn't affect game once initialized, so same as above